### PR TITLE
AV-56105: Add Server 7.2 Features to Capella docs

### PR DESCRIPTION
**Pull request by simon-dew**
This PR adds time series documentation to the Capella branch. I had problems getting the `cbimport` tool to work with Capella, so for now I have replaced the import data examples with instructions using the Import tab in the GUI.

Preview build:

* [Time Series Data](https://simon-dew.github.io/docs-site/release-72/cloud/n1ql/n1ql-language-reference/time-series.html)
* [_TIMESERIES Function](https://simon-dew.github.io/docs-site/release-72/cloud/n1ql/n1ql-language-reference/timeseries.html)

Note that merging this PR will make the time series documentation appear in the Capella documentation immediately. Don't merge until Server 7.2 for Capella is released.

Docs issue: AV-56105

### CODE REVIEW

File: modules/n1ql/pages/n1ql-language-reference/time-series.adoc [hunk 1]
```diff
@@ -0,0 +1,535 @@
...
+:datefun: {n1ql-language-reference}/datefun.adoc
+
+[abstract]
+{description}
+
+Time series data is any data which changes over time.
+In the case of data management, it refers to data collected frequently, in regular or irregular intervals, from a device or a process.
+The changing data is typically numerical, and changes incrementally.
+
+IMPORTANT: This feature is only available on clusters using Couchbase Server 7.2 or later.
```
Comment (dgnicholson) [inline modules/n1ql/pages/n1ql-language-reference/time-series.adoc]:
```suggestion
IMPORTANT: This feature is only available on databases using Couchbase Server 7.2 or later.
```

File: modules/n1ql/pages/n1ql-language-reference/timeseries.adoc [hunk 2]
```diff
@@ -0,0 +1,665 @@
...
+
+{description}
+
+The time series data must be stored in Couchbase time series documents.
+For more information, see {time-series}[].
+
+The function dynamically generates data point objects from the array of time series data in the input document.
+For a regular time series, the function uses the time series interval to generate a date-time stamp for each time point automatically.
+
+IMPORTANT: This feature is only available on clusters using Couchbase Server 7.2 or later.
```
Comment (dgnicholson) [inline modules/n1ql/pages/n1ql-language-reference/timeseries.adoc]:
```suggestion
IMPORTANT: This feature is only available on databases using Couchbase Server 7.2 or later.
```

**Review by dgnicholson — APPROVED** [review]
Thanks @simon-dew!
LGTM, just two small suggestions.

File: modules/n1ql/pages/n1ql-language-reference/time-series.adoc (hunk 1 above)
Comment (simon-dew) [reply modules/n1ql/pages/n1ql-language-reference/time-series.adoc]: Done

File: modules/n1ql/pages/n1ql-language-reference/timeseries.adoc (hunk 2 above)
Comment (simon-dew) [reply modules/n1ql/pages/n1ql-language-reference/timeseries.adoc]: Done

**Review by dgnicholson — APPROVED** [review]

**Review by anand-gl — APPROVED** [review]

**Comment by kkchbase1** [comment]
I think instead of replicating the server content, we should simply have pointers here so when there are improvements in time series functions, they dont have to be replicated/copied here in Capella

**Pull request merged**